internal/querynode/shard_cluster.go

```diff
@@ -1106,3 +1107,57 @@ func (sc *ShardCluster) GetSegmentInfos() []shardSegmentInfo {
 func (sc *ShardCluster) getVersion() int64 {
 	return sc.version
 }
+
+func getSearchWithStreamingFunc(searchCtx context.Context, req *querypb.SearchRequest, node *QueryNode, qs *queryShard, nodeID int64, streamingResults **internalpb.SearchResults) withStreaming {
```

Comment:
Please do not use `**internalpb.SearchResults`. We could modify `WithStreaming` signature to fix this problem